xml/System.Runtime.CompilerServices/SwitchExpressionException.xml

```diff
@@ -88,8 +88,8 @@
         <Parameter Name="unmatchedValue" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="unmatchedValue">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="unmatchedValue">The switch value that does not match any switch cases.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Runtime.CompilerServices.SwitchExpressionException" /> class with the specified message.</summary>
```

Comment:
This isn't the correct description; it probably should be "Initializes...with an unmatched switch value."